qchem/pennylane_qchem/qchem/__init__.py

```diff
@@ -18,14 +18,14 @@
 
 __all__ = [
     "read_structure",
-    "meanfield_data",
+    "meanfield",

```

Comment:
Are these pylint errors expected?

Reply:
These error messages seem to be gone now.